camel/agents/chat_agent.py

```diff
@@ -79,6 +79,10 @@
     from camel.utils import track_agent
 
 
+class ModelProcessingError(Exception):
+    """Raised when an error occurs during model processing."""
```

Comment:
Please update the docstring by following the unified format we are using today. Thanks

Reply:
Updated